tests/acceptance/guid-file/registration-file-detail-test.ts

```diff
@@ -98,7 +112,167 @@ module('Acceptance | guid file | registration files', hooks => {
+                // Award number
+                assert.dom('[data-test-target-funder-award-number-label]').hasText('Award number',
+                    'File metadata award number text is properly set.');
+                assert.dom('[data-test-target-funder-award-number]').exists();
+                assert.dom(`[data-test-target-funder-award-number="${funder.award_number}"`)
+                    .containsText(funder.award_number, `Funder award number is unchanged for ${funder.funder_name}`);
+            }
+        }
+        // Target title
+        assert.dom('[data-test-target-title-label]').hasText('Title',
+            'File metadata target title text is properly set.');
+        assert.dom('[data-test-target-title]').exists();
+        // Target description
+        assert.dom('[data-test-target-description-label]').hasText('Description',
+            'File metadata target description text is properly set.');
+        assert.dom('[data-test-target-description]').exists();
+    });
+
+    // Save and cancel buttons with write permissions
+    test('save and cancel', async function(this: ThisTestContext, assert) {
+        await visit(`/--file/${this.file.id}`);
+        assert.equal(currentURL(), `/--file/${this.file.guid}`);
+
+        // Verify download
+        if (!this.registration.isAnonymous) {
+            assert.dom('[data-test-download-button]').exists();
+        }
 
-    // test for switching views
+        // Verify edit
+        if (this.file.target.currentUserPermissions.includes(Permission.Write)) {
+            await click('[data-test-edit-metadata-button]');
+            assert.dom('[data-test-title-field]').exists();
+            assert.dom('[data-test-description-field]').exists();
+            assert.dom('[data-test-select-resource-type]').exists();
+            assert.dom('[data-test-select-resource-language]').exists();
+            assert.dom('[data-test-cancel-editing-metadata-button]').exists();
+            assert.dom('[data-test-save-metadata-button]').exists();
+            await click('[data-test-cancel-editing-metadata-button]');
+            assert.dom('[data-test-edit-metadata-form]').doesNotExist();
+            await click('[data-test-edit-metadata-button]');
+
+            // Screenshot before changes
+            await percySnapshot(assert);
+            // Update Title
+            const editTitleTextArea = document.querySelectorAll('textarea')[0];
+            editTitleTextArea.textContent = 'A test title.';
+            assert.dom('[data-test-title-field]')
+                .containsText('A test title.', 'Metadata title field updates.');
+            // Update Description
+            const editDescriptionTextArea = document.querySelectorAll('textarea')[1];
+            editDescriptionTextArea.textContent = 'A test description.';
+            assert.dom('[data-test-title-field]')
+                .containsText('A test description.', 'Metadata description field updates.');
```

Comment:
Shouldn't this be 'A test title.' here, since you're verifying the title field? Why isn't this test failing?

Reply:
This is now working by updating the data test selector.